src/Cases.py

```diff
@@ -90,6 +90,9 @@ def get_input_specs(cls):
     debug.addSub(InputData.parameterInputFactory('macro_steps', contentType=InputTypes.IntegerType,
         descr=r"""sets the number of macro steps (e.g. years) the stochastic synthetic histories and dispatch
               optimization should include. \default{1}"""))
+    debug.addSub(InputData.parameterInputFactory('dispatch_plot', contentType=InputTypes.BoolType,
+        descr=r"""provides a dispatch plot after running through \xmlNode{inner_samples} and
+              \xmlNode{macro_steps} provided. \default{True}"""))
```

Comment:
It may be worth clarifying that the user has to set this to "false" to not get a plot; otherwise, they're getting one. This should be clear from the default, admittedly.

Reply:
Sure I will add something to that effect.